core/modules/link/link.module

```diff
@@ -420,6 +439,20 @@ function _link_validate(&$item, $delta, $field, $entity, $instance, $langcode, &
         'error_element' => array('url' => FALSE, 'title' => TRUE),
       );
     }
+    // Validate the URL type.
+    if ($instance['settings']['type'] != LINK_BOTH) {
```

Comment:
I guess, this should only happen, ` if ($instance['settings']['validate_url']) {...`, see the code block above.

Reply:
Disagree, as per discussion in https://github.com/backdrop/backdrop-issues/issues/5678.